### DISCUSSION TITLE
Allow to configure filetype

### DISCUSSION THREAD

**Discussion by gowep** [NeogitOrg/neogit#215] [Ideas]
Hello, 
I saw that you chaged the commit filetype in https://github.com/TimUntersberger/neogit/commit/33ccd4a1c84cc4c792ceb2832bd18a9071eaba1c.
Could you make this optional or a config option?
Right now I use gitcommit as a filetype for lots of settings (spellcheck, disable indentline) and configuring/maintaining this for the new filetype would double the work.

**Comment by TimUntersberger** [comment]
I don't think you should have to duplicate your code. Have you tried the below `ftplugin`?

ftplugin/NeogitCommitMessage.vim
```vim
runtime ftplugin/gitcommit.vim
```

**Comment by gowep** [comment]
True that would fix a lot of duplication. But then theres still the problem of plugin settings, e.g.
filetype_exclude for indentline.

**Comment by gowep** [comment]
Thanks for your help, I am using after/ftplugin/NeogitCommitMessage.vim now and that works well for me.
